src/test/java/resources/voucherManagementTest.feature

```diff
@@ -74,32 +85,29 @@ Feature: Voucher Management Api Test
     When I call the redeem voucher API with expected status of 400
     And I should be able to assert the redeem voucher validation for negative response
 
-  @gov @createAndActivateVoucher @redeemVoucherFailure
+  @gov
   Scenario: VR-005 Redeem Voucher Api Negative Test when voucher is already redeemed
+    Given I can create and activate a voucher
     Given I can create an RedeemVoucherRequestDTO for voucher redemption
     When I call the redeem voucher API with expected status of 200
+    Then I check for redeem voucher failure
 
  @gov
   Scenario: Unsupported Parameter Validation for Create Voucher API test
     Given I can create an VoucherRequestDTO for voucher creation with unsupported parameter parameter
     When I call the create voucher API with expected status of 400 and stub "/createVoucher"
     Then I should be able to assert the create voucher validation for negative response
 
-#  @gov @createAndActivateVoucher
-#  Scenario: Redeem Voucher Api Test
-#    Given I can create an RedeemVoucherRequestDTO for voucher redemption
-#    When I call the redeem voucher API with expected status of 200
-#    Then I can assert that redemption was successful by asserting the status in response
-
-  @gov @createAndActivateVoucher
+  @gov
    Scenario: Conflicting/unique data validations check for Create Voucher API
+    Given I can create and activate a voucher
     Then I will call the fetch voucher API with expected status of 200
     When I call the create voucher API with expected status of 409 and stub "/createVoucher"
 
-  @Ignore
-  Scenario: Create a csv file for voucher number and voucher serial number
-    When I can inject MockServer
-    Then I can start mock server
-    And I can register the stub with "/createVoucher" endpoint for "PUT" request with status of 200
-    And I can register the stub with "/activateVoucher" endpoint for "PUT" request with status of 200
-    When I call the create, Activate voucher API and store it in "vouchertest/loadTest_demo.csv"
+#  @Ignore
```

Comment:
@ignore should ignore this from the run. 
do we need to comment them?

Reply:
Yes, will fix it.